src/common/components/hosts/DiskRole.tsx

```diff
@@ -81,9 +82,9 @@ const DiskRoleDropdown: React.FC<DiskRoleDropdownProps> = ({
         // TODO(mlibra): Improve for the case onDiskRole === undefined
         setOpen(false);
       };
-      asyncFunc();
+      void asyncFunc();
```

Comment:
what is the reason for adding `void` here ?

Reply:
> This operator allows evaluating expressions that produce a value into places where an expression that evaluates to undefined is desired.
Since we ignore the returned promise, IMHO it makes sense to use it. The code gets executed as usual, the promise still fulfills or rejects anyway.

Reply:
You can think of it as an explicit way of saying: "just call this (async) function, I (explicitly) don't care about the outcome"

Reply:
so without `void` it means something different than `"just call this (async) function, I (explicitly) don't care about the outcome"` ?

Reply:
If you and I have the same dev environment setup, you should see an annoying linting message saying that you're ignoring the value returned by the promise and then the alternative will be to silent the linter for this line or just use the void operator as I did.

Reply:
linter setup is defined by `.eslintrc` in root folder, isnt it ? Why do you use different setup ? The linter validation should be the same as CLI config otherwise we start adding random stuff (like this `void`) and later somebody can remove it because our CLI linter config changes and it starts to complain etc. Endless loop.

Reply:
I totally agree, with you. My setup consumes the eslint configuration from `.eslintrc` located at the root folder. I'm wondering whether you see this message about the returned promise being ignored, as I do, on this line or not?

Reply:
I dont.
Even running `yarn test:lint` does not show any warnings

Reply:
![image](https://user-images.githubusercontent.com/77008341/139655007-16051175-4ae1-4ce3-b431-b48a704c8d45.png)

Reply:
Is this really an eslint rule ? I could not find anything like that.
Isnt it just some smartness build in IntelliJ ?

Reply:
Yep seems it's coming from inteliJ. Is that a bad thing?

Reply:
I guess its fine. I was mostly worried that you have different eslint config. 
If you dont mind putting void on other places in the lib in the future (as people who do not use InteliJ wont do it), instead of turning that inspection info off, then Im good :)

Reply:
> If you don't mind putting void on other places in the lib in the future

I'll even do that with pleasure :)

Reply:
I still think it's just about IDE setup. Such syntactic sugar just makes the code/language talkative, harder to read.

Reply:
Well, this is JavaScript @mareklibra. May I ask what part exactly do you find hard to read, maybe I can explain it to you?

Reply:
My point is not in that. I was just referring to style - to avoid whatever is not needed.

Reply:
[There's an interesting discussion in github about this ](https://stackoverflow.com/questions/41278900/intellij-idea-warning-promise-returned-is-ignored-with-aysnc-await)
I'm assuming Joni saw this...
The solution Joni mentioned is one of the answers.
I thought the solution of adding .catch might also be a good option, since error handling is always a good idea...

Reply:
`try catch` makes more sense to me than adding `void`.

Reply:
@rawagner you cannot use a try-catch there, `asyncFunc` is not being called inside an async block.
What you can do is actually something like this: `asyncFunc().catch(/*...*/);`